e2e/test_operator/operator_test.go

```diff
@@ -1747,4 +1747,150 @@ var _ = Describe("Operator", Label("e2e", "pr"), func() {
 			Expect(cluster.Status.DatabaseConfiguration.RoleCounts.Proxies).NotTo(BeZero())
 		})
 	})
+
+	FWhen("running with tester processes", func() {
```

Comment:
Do we need to clear the focus before merging?

Reply:
Thanks for catching that 👍